src/gcs/gcsBrowserWidget.tsx

```diff
@@ -102,10 +102,14 @@ export class GcsBrowserWidget extends Widget {
             bucket: path.bucket
           });
 
-          if(content.items && content.items.length > 0) {
+          if (content.items && content.items.length > 0) {
             const result = await showDialog({
               title: 'Upload files',
-              body: file.name + ' already exists in ' + path.bucket +' and click ok to overwriting file',
+              body:
+                file.name +
+                ' already exists in ' +
+                path.bucket +
+                ' and click ok to overwriting file',
```

Comment:
Please modify the message 'and click ok to overwriting file' to: 'Do you want to overwrite the file?'

Reply:
Resolved